metabrainz/templates/index/team.html

```diff
@@ -327,7 +327,7 @@ <h4>{{ _('<span class="name">David "drsaunde" Saunders</span> &mdash; MusicBrain
       <img
         alt="MonkeyPython's photo"
```

Comment:
Do you want to change this (the alt text of the image, only appears if image loading fails) as well @ApeKattQuest-MonkeyPython ?

```suggestion
        alt="Akira Holm's photo"
```